### ISSUE TITLE
Survivality (1.2.2) crashes with Lithium

### ISSUE THREAD

**Issue by CreepahJacket** [jack-zisa/survivality#2]
## Preface
it worked fine when i tried it with 1.2.1, text below is auto-generated
## Stacktrace
```
org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: @At("INVOKE") on net/minecraft/class_1297::survivality_removeBoatWaterlilyCollisions with priority 1000 cannot inject into net/minecraft/class_1297::method_20736(Lnet/minecraft/class_1297;Lnet/minecraft/class_243;Lnet/minecraft/class_238;Lnet/minecraft/class_1937;Ljava/util/List;)Lnet/minecraft/class_243; merged by me.jellysquid.mods.lithium.mixin.entity.collisions.movement.EntityMixin with priority 1000 [PREINJECT Applicator Phase -> survivality.mixins.json:entity.EntityMixin from mod survivality -> Prepare Injections ->  -> redirect$bca000$survivality$removeBoatWaterlilyCollisions(Lnet/minecraft/class_1937;Lnet/minecraft/class_1297;Lnet/minecraft/class_238;)Ljava/lang/Iterable; -> Prepare]
	at org.spongepowered.asm.mixin.injection.code.Injector.findTargetNodes(Injector.java:305)
	at org.spongepowered.asm.mixin.injection.code.Injector.find(Injector.java:240)
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.prepare(InjectionInfo.java:421)
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.prepareInjections(MixinTargetContext.java:1338)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.prepareInjections(MixinApplicatorStandard.java:1053)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:395)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:327)
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:421)
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:403)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:422)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:323)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:112)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
	at net.minecraft.class_2378.<clinit>(class_2378.java:207)
	at net.minecraft.class_2966.method_12851(class_2966.java:50)
	at net.minecraft.client.main.Main.method_44604(Main.java:161)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:462)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:88)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:126)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:71)

```

## Mod list
```
- fabric-api 0.76.0+1.19.2
- fabricloader 0.14.19
- java 18
- lithium 0.11.1
- minecraft 1.19.2
- mixin-conflict-helper 1.2.0
- modmenu 4.2.0-beta.2
- survivality 1.2.2
```

## Game info
Minecraft Version: 1.19.2
Fabric Loader Version: 0.14.19

*Message autogenerated using [MixinConflictHelper](https://github.com/isXander/MixinConflictHelper)*

**Comment by jack-zisa** [comment]
I released 1.2.3 addressing this issue. I had success testing it, but let me know if it is fixed for you!

**Comment by CreepahJacket** [comment]
can confirm that it's fixed 👍
